app/builders/account_builder.rb

```diff
@@ -94,8 +94,8 @@ def domain_blocked?
 
   def blocked_domains
     domains = GlobalConfigService.load('BLOCKED_EMAIL_DOMAINS', '')
-    domains.split("\n").map(&:strip) if domains.present?
+    return [] if domains.blank?
 
-    []
+    domains.split("\n").map(&:strip) if domains.present?
```

Comment:
```suggestion
    domains.split("\n").map(&:strip)
```

Can avoid `.present` check as we did check for `.blank?` above